colour/models/rgb/ycbcr.py

```diff
@@ -0,0 +1,359 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Y'CbCr Colour Encoding
+======================
+
+Defines the Y'CbCr encoding (Y'CbCr is not a colour space) transformations:
+
+-   :func:`rgb_to_YCbCr`
+-   :func:`YCbCr_to_rgb`
+-   :func:`rgb_to_YcCbcCrc`
+-   :func:`YcCbcCrc_to_rgb`
+
+References
+----------
+.. [1]  Wikipedia. YCbCr. Retrieved February 29, 2016, from
```

Comment:
I'll need to put all those references in our reference manager for the Bibliography output.